tests/core/resource/test_filter.py

```diff
@@ -219,28 +219,35 @@ def test_exclude_returns_400_if_value_has_wrong_type(self):
         self.assertRaises(httpexceptions.HTTPBadRequest,
                           self.resource.collection_get)
 
-    def test_contains_with_strings(self):
+    def test_contains_can_filter_with_one_string(self):
+        self.validated['querystring'] = {'contains_colors': ["red"]}
+        result = self.resource.collection_get()
+        values = [item['colors'] for item in result['data']]
+        for value in values:
+            assert "red" in value
```

Comment:
These tests are each trivially passable by returning an empty list. Maybe it would be good to add an assertion counting how many objects are returned, or verifying that any object which wasn't returned *didn't* have "red".